nouvelles-perspectives-en-sciences-sociales.csl

```diff
@@ -24,10 +26,10 @@
       <term name="translator" form="verb-short">trad. par</term>
       <term name="translator" form="short">traduction</term>
       <term name="interviewer" form="verb">entretien réalisé par</term>
-      <term name="in">dans </term>
+      <term name="in">dans&#160;</term>
       <term name="edition">édition</term>
       <term name="accessed">consulté le</term>
-      <term name="at">disponible sur :</term>
+      <term name="at">disponible sur&#160;:</term>
```

Comment:
approved

Reply:
This modification will not affect our proposition ? We need a bibliography for our journal as the sample citations that csl-bot sent us. The citations generated and based on our proposed changes.